### PR TITLE
added -is command

### PR DESCRIPTION
**Pull request by anveshthakur**
Hi @osteensco, 

Added -is command and tested it. It is working fine. I have also written a test case for it. 

This solves issue #18 

Thanks.

### CODE REVIEW

File: ft/commands.go [hunk 1]
```diff
@@ -274,6 +276,36 @@ func showVersion(data *CmdArgs) error {
...
+		fmt.Println("This command is not supported on your os")
+	}
+
+	pathMaps := data.allPaths
+
+	for k := range pathMaps {
+		v, _ := pathMaps[k]
+
+		if strings.Compare(v, dir) == 0 {
+			fmt.Println(strings.Trim(k, "\n"))
```
Comment (osteensco) [inline ft/commands.go:300]: I'd rather this print something clear for the user like "[path] is saved to key [name of key]".

File: ft/commands.go [hunk 2]
```diff
@@ -274,6 +276,36 @@ func showVersion(data *CmdArgs) error {
...
+			return nil
+		}
+	}
+
+	fmt.Println("No key found for this path!")
```
Comment (osteensco) [inline ft/commands.go:305]: similarly, this should be more explicit as well.

**Review by osteensco — CHANGES_REQUESTED** [review]

File: ft/commands.go [hunk 3]
```diff
@@ -274,6 +276,36 @@ func showVersion(data *CmdArgs) error {
 	return nil
 }
 
+func showDirectoryVar(data *CmdArgs) error {
```
[the remark this answers is not in the document]
Comment (anveshthakur) [reply ft/commands.go:279]: Changed to get the CWD from os package

File: ft/commands.go (hunk 1 above)
Comment (anveshthakur) [reply ft/commands.go:300]:
```sh
Directory /home/go/fastTravelCLI is saved to key : fast
```

I have changed output string to this.

**Review by osteensco — APPROVED** [review]

**Pull request merged**